lib/blueprinter/base.rb

```diff
@@ -37,12 +37,14 @@ class Base
     #   end
     #
     # @return [Field] A Field object
-    def self.identifier(method, name: method, extractor: AutoExtractor.new)
-      view_collection[:identifier] << Field.new(method, name, extractor, self)
-    end
-
-    def self.inherited(subclass)
-      subclass.send(:view_collection).inherit(view_collection)
+    def self.identifier(method, name: method, extractor: AutoExtractor.new, &block)
```

Comment:
Would you mind adding to the method documentation for this and adding it to the README similar to how you did for the associations?
https://github.com/procore/blueprinter/blob/f849049fedf7adea5f13d11b2514b6c8b54f7efa/lib/blueprinter/base.rb#L124-L139
https://github.com/procore/blueprinter/blob/master/README.md#defining-an-association-directly-in-the-blueprint